src/test/java/com/iexec/core/security/JwtTokenProviderTests.java

```diff
@@ -73,6 +78,21 @@ void isValidTokenTrue() {
         assertThat(isValidToken).isTrue();
     }
 
+    @Test
+    void isValidTokenFalseSinceExpired() {
+        when(challengeService.getChallenge(WALLET_ADDRESS)).thenReturn(" challenge");
+        String token = jwtTokenProvider.createToken(WALLET_ADDRESS);
+        new DefaultClaims().setAudience(WALLET_ADDRESS);
+        try (MockedStatic<Jwts> mockedStatic = mockStatic(Jwts.class)) {
```

Comment:
Are we forced to mock this static class? Can't we forge an expired token?

Reply:
not really easy, need to inject the encryption key and the forged token in JwtTokenProvider

Reply:
Alright. Let's keep it as is then :)